docs/deployment/feature_gates.md

```diff
@@ -33,6 +33,8 @@ The following tables are a summary of the feature gates that you can set on diff
 | OpenTelemetryCollector                   | `false` | `Alpha` | `1.124` |         |
 | UseUnifiedHTTPProxyPort                  | `false` | `Alpha` | `1.130` |         |
 | VPAInPlaceUpdates                        | `false` | `Alpha` | `1.133` |         |
+| CustomDNSServerInNodeLocalDNS            | `true`  | `Beta`  | `1.133` |         |
+
```

Comment:
Cosmetic: There is one additional trailing empty line. There was no such line in the suggestion in https://github.com/gardener/gardener/pull/13511#discussion_r2549920016.

```suggestion
```